README.md

```diff
@@ -76,9 +72,8 @@ You can [download](https://projectnaomi.com/download) latest installable version
 
 This software uses code from the initial Jasper Project & the Jasper2fork.
 
-- [JasperProject](http://jasperproject.github.io/)
-- [Jasper2Fork](https://github.com/andweber/j2f)
```

Comment:
Note: Naomi project was previously named J2F (Jasper2fork), andweber's repository is just a fork of it.

Reply:
Ok, that section did not change. Just fixing the markdown list style.